image-builder/image_builder.sh

```diff
@@ -38,299 +56,373 @@ Options:
 	-r Free space of the root partition in MB ENV: ROOT_FREE_SPACE
 
 Extra environment variables:
-	AGENT_BIN:  use it to change the expected agent binary name
-	AGENT_INIT: use kata agent as init process
-	DAX: If 'yes' will build the image with DAX support. The first 2 MB of the
-	     resulting image are reserved for the device namespace information
-	     (metadata) that is used by the guest kernel to enable DAX.
+	AGENT_BIN:  Use it to change the expected agent binary name
+	AGENT_INIT: Use kata agent as init process
+	FS_TYPE:    Filesystem type to use. Only xfs and ext4 are supported.
```

Comment:
Sorry I dont get why xfs is being added

Reply:
DAX support xfs and ext4

Reply:
I'd add Only `xfs and ext4 are supported for DAX support`

Reply:
the current image builder implementation only support `ext4` because `mkfs.ext4` is hardcoded. In this PR I'm adding support for `xfs`, so only these two format will be supported. If we want to support more formats then we have to use the different versions of `mkfs.*` and that is out of the scope of this PR.